applications/portal/frontend/src/App.tsx

```diff
@@ -21,7 +21,9 @@ const App = () => {
         <CssBaseline />
         <Navbar />
         <Switch>
-          <Route exact path="/" component={Home} />
+          <Route exact path="/">
+            <Home activeTab="all results" />
```

Comment:
It would be better to use constants for tab names

Reply:
done